.circleci/ansible/neon-stress.hosts

```diff
@@ -12,6 +12,7 @@ pageservers
 safekeepers
```

Comment:
Since `ansible` files are now only used in GH Actions, we could move it and remove the whole `.circleci` directory (if we move the `release` deploys too, as suggested)